frontend/src/index.css

```diff
@@ -858,3 +858,7 @@ Other button classes are defined further down together with other classes for th
   height: 3em;
   width: 8em;
 }
+
+.group-select-wrapper {
+  margin: 3em;
+}
```

Comment:
Is there a certain reason you used `em` here? The unit is relative to the font-size of the parent element, which makes it slightly unpredictable in my experience. It's usually better to use `rem` instead, which is relative to the font-size of the root element, which is the `<html>` element, so it's more consistent. 
I see now that `em` is used even in classes above here, that are not part of this PR, but I think we should try to avoid it.